templates/layouts/configs/_footer.html

```diff
@@ -10,11 +15,7 @@
 
   {% set contact_us_url = "https://cyfrifiad.gov.uk/cysylltu-a-ni/" %}
   {% set cookies_url = "https://cyfrifiad.gov.uk/cwcis/" %}
-
-{% else %}
-
-  {% set contact_us_url = "https://census.gov.uk/contact-us/" %}
-  {% set cookies_url = "https://census.gov.uk/cookies/" %}
+  {% set help_url = "https://cyfrifiad.gov.uk/help/sut-i-ateb-y-cwestiynau/help-y-cwestiynau-ar-lein-ew/" %}
```

Comment:
This isn't working if you're on one of the static pages in eQ:
- Start the Welsh household survey in Welsh
- Click on the `/privacy` link (Preifatrwydd a diogelu data)
- The links to Contact Us, Help and Cookies are the English links

Reply:
Ive fixed this now. I for some reason language_code wasn't being set on static pages so I have added it to the context and got it from there